worf/views/create.py

```diff
@@ -9,27 +9,19 @@ def serialize(self):
         return {}
 
     def post(self, request, *args, **kwargs):
+        serializer = self.get_serializer()
 
-        # Deprecate ------------------------------------------------------------
```

Comment:
The reason this code needs to stay is because everything that is not using a serializer will break without it

Reply:
I trust you noticed that everything now uses a serializer